apps/desktop/desktop/src/main/java/bisq/desktop/main/content/mu_sig/offerbook/MuSigOfferbookController.java

```diff
@@ -226,8 +231,24 @@ public void clear() {
         });
         model.getSortedMarketItems().setComparator(model.getSelectedMarketSortType().get().getComparator());
 
+        Direction persistedOfferlistFilter = settingsService.getCookie().asString(CookieKey.MU_SIG_OFFERLIST_FILTER)
+                .map(name -> ProtobufUtils.enumFromProto(Direction.class, name, null))
+                .orElse(null);
+        model.getSelectedMuSigOfferlistFilter().set(persistedOfferlistFilter);
+        selectedOfferlistFilterPin = EasyBind.subscribe(model.getSelectedMuSigOfferlistFilter(), filter -> {
+           if (filter == null) {
+               model.setMuSigOffersFilterPredicate(item -> true);
+               settingsService.setCookie(CookieKey.MU_SIG_OFFERLIST_FILTER, "all offers"); // fallback will be null
```

Comment:
Should be null as that is the value expected for all offers. But better use a distinct enum.